### PR TITLE
Cleanup global styles & main menu

### PR DESCRIPTION
**Pull request by craftablescience**
- Clean up global styles
- Clean up main menu
- Run eslint on `new-menu` commits and PRs

### CODE REVIEW

File: styles/base/utility.sass [hunk 1]
```diff
@@ -1,166 +1,181 @@
...
+
+
+.debug-background
+	background-color: rgba(255, 0, 255, 0.5)
+
+// ================
+// ANIMATIONS
+// ================
+
+@keyframes fadein
```
Comment (Trico-Everfire) [inline styles/base/utility.sass:171]: Per style guide animations must be camelCase.

File: styles/base/utility.sass [hunk 2]
```diff
@@ -1,166 +1,181 @@
...
+	0%
+		opacity: 0.0
+	100%
+		opacity: 1.0
+
+@keyframes fadeout
```
Comment (Trico-Everfire) [inline styles/base/utility.sass:177]: ditto

**Review by Trico-Everfire — CHANGES_REQUESTED** [review]
for the rest it all looks good.

File: scripts/common/gameui.js [hunk 3]
```diff
@@ -1,10 +1,11 @@
-"use strict"
+/* eslint-disable no-unused-vars */
+"use strict";
 
 var GAME_UI_STATE = {
```
[the remark this answers is not in the document]
Comment (craftablescience) [reply scripts/common/gameui.js:4]: nope, `const` messes with reloading (f7/f8)

File: styles/base/utility.sass (hunk 1 above)
Comment (craftablescience) [reply styles/base/utility.sass:171]: good catch, thx!

File: styles/base/utility.sass (hunk 2 above)
Comment (craftablescience) [reply styles/base/utility.sass:177]: will fix these when i can

File: scripts/common/gameui.js (hunk 3 above)
[the remark this answers is not in the document]
Comment (Trico-Everfire) [reply scripts/common/gameui.js:4]: all right, I'll resolve this then.

File: layout/tooltips/tooltip_base.xml [hunk 4]
```diff
@@ -1,6 +1,6 @@
 <root>
 	<styles>
-    <include src="file://{resources}/styles/chaosstyles.sass" />
+    <include src="file://{resources}/styles/common.sass" />
```
Comment (AWildErin) [inline layout/tooltips/tooltip_base.xml:3]: indenting

File: layout/tooltips/tooltip_text.xml [hunk 5]
```diff
@@ -1,6 +1,6 @@
 <root>
 	<styles>
-    <include src="file://{resources}/styles/chaosstyles.sass" />
+    <include src="file://{resources}/styles/common.sass" />
```
Comment (AWildErin) [inline layout/tooltips/tooltip_text.xml:3]: Indenting

File: layout/tooltips/tooltip_title_image_text.xml [hunk 6]
```diff
@@ -1,6 +1,6 @@
 <root>
 	<styles>
-    <include src="file://{resources}/styles/chaosstyles.sass" />
+    <include src="file://{resources}/styles/common.sass" />
```
Comment (AWildErin) [inline layout/tooltips/tooltip_title_image_text.xml:3]: Indenting

File: layout/tooltips/tooltip_title_text.xml [hunk 7]
```diff
@@ -1,6 +1,6 @@
 <root>
 	<styles>
-    <include src="file://{resources}/styles/chaosstyles.sass" />
+    <include src="file://{resources}/styles/common.sass" />
```
Comment (AWildErin) [inline layout/tooltips/tooltip_title_text.xml:3]: Ditto

File: styles/base/panorama_base.sass [hunk 8]
```diff
@@ -34,10 +34,10 @@ TextTooltip
 	font-size: 12px
 	color: #111111ff
 
-    visibility: collapse
+	visibility: collapse
 
 TextTooltip.TooltipVisible
-    visibility: visible
+	visibility: visible
```
Comment (AWildErin) [inline styles/base/panorama_base.sass:40]: Ditto

**Review by AWildErin — APPROVED** [review]
Style issues mainly, not urgent to be solved, but would be appreciated

File: styles/base/panorama_base.sass (hunk 8 above)
Comment (AWildErin) [reply styles/base/panorama_base.sass:40]: Now I'm off mobile I can see that this is perfectly fine. GitHub sucks with diff rendering some times

**Review by AWildErin — APPROVED** [review]

**Pull request merged**